### PR TITLE
best hr events in nl 2018 blog post draft

### PR DESCRIPTION
**Pull request by hunterwright696**
draft blog post for best hr events in nl for 2018. also includes international hr events in london, birmingham, and barcelona.

### CODE REVIEW

**Comment by emmalou** [comment]
@kdamball when I click on this View Deployment, it doesn't seem to load. Is there a reason? Could it have timed out?

**Comment by kdamball** [comment]
@emmalou it was a while since it was created (>2wks). I recreated it - it should be live in the next 10mins or so

**Comment by emmalou** [comment]
@kdamball thank you! For reference how can I recreate it?

**Comment by hunterwright696** [comment]
Added a **commit** titled 'changes to netherlands hr events blog post 12/24' **fixing issues** with conclusion and non-NL HR events

File: _posts/2017-12-05-the-best-hr-events-in-the-netherlands.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+subtitle:
+date:    2018-1-21 09:31:53
+layout:  post
+author:  hunter_wright
+categories: HR Tips
+cover_image: hr-netherlands.png
+permalink:  the-best-hr-events-in-the-netherlands/
+importance: 2
+---
+The Netherlands has dozens of great HR events. Check out which made our list of best HR events in 2018.
```
Comment (anamiruna) [inline _posts/2017-12-05-the-best-hr-events-in-the-netherlands.md]: We've put together a curated list of best HR events in 2018. Without further ado, here it is:

Comment (hunterwright696) [reply _posts/2017-12-05-the-best-hr-events-in-the-netherlands.md]: changes made!

**Pull request merged**